components/select/index.tsx

```diff
@@ -4,12 +4,15 @@ import * as React from 'react';
 import omit from 'omit.js';
 import classNames from 'classnames';
 import RcSelect, { Option, OptGroup, SelectProps as RcSelectProps } from 'rc-select';
+import OptionProps from 'rc-select/lib/Option';
 import { ConfigConsumer, ConfigConsumerProps } from '../config-provider';
 import getIcons from './utils/iconUtil';
 import SizeContext, { SizeType } from '../config-provider/SizeContext';
 
 type RawValue = string | number;
 
+export type OptionProps = typeof OptionProps;
```

Comment:
What about:

```suggestion
export type OptionProps;
```

Reply:
It is because it was giving me an error of eslint: 
**Parsing error: '=' expected** also it didn't allow me to push the code. So this is the reason why I write this.

Reply:
Try this~

```
export { OptionProps } from 'rc-select/lib/Option';
```

Reply:
like https://github.com/ant-design/ant-design/commit/14e2a56b757f11b84ab947c58fb86d6ebb2c8f4a

Reply:
Facing these issues:

**case 1:**
export { OptionProps } from 'rc-select/lib/Option';
**`Export declaration conflicts with exported declaration of 'OptionProps'.`**
export type OptionProps;
**Parsing error: '=' expected.eslint**

**case 2:**
export { OptionProps } from 'rc-select/lib/Option';
**`Export declaration conflicts with exported declaration of 'OptionProps'.`**
export OptionProps;
**OptionProps' only refers to a type, but is being used as a value here.ts(2693)**